dev-libs/spdlog/files/spdlog-libfmt-10.0.0.patch

```diff
@@ -0,0 +1,30 @@
+Bug: https://bugs.gentoo.org/906069
+Upstream: https://github.com/gabime/spdlog/pull/2694
+
+Fixing spdlog-1.11.0 fails compile with libfmt-10.0.0
+
+diff --git a/include/spdlog/common.h b/include/spdlog/common.h
```

Comment:
thanks, I added some more metadata by going to the commit link then appending .patch. Please do that in future

thank you!

Reply:
OK, I usually delete metadata of the patch , only keep the code part...
I saw your push f6ca1358c59588198f05aa99b61fdac26dfad42b , thanks !